api/src/main/proto/stellarstation/api/v1/monitoring/monitoring.proto

```diff
@@ -231,6 +231,30 @@ message AntennaState {
   antenna.AntennaPolarization polarization = 3;
 }
 
+// The current state of the plan being executed by the ground station.
+message PlanState {
+  // The status of the plan.
+  enum Status {
+    UNKNOWN = 0;
+    STARTED = 1;
```

Comment:
So what's the difference between `STARTED` and `EXECUTING`? This message is a state, not an event or something.

Reply:
I initially had only EXECUTING then I added STARTING which became STARTED. 

It's true that as a state, there isn't a difference. I thought it would be clean to have STARTED as a state which is only sent once and it must be sent as soon as the starpass starts processing.

I thought that EXECUTING could be interpreted as being sent at any time (although we could require that it is sent immediately when processing starts too).

Do you prefer to get rid of STARTED?

Reply:
I have no opinion on whether there should be `STARTED` or something, but if there is I prefer `PREPARING` or `SETTINGUP`. When I saw these enums for the first time I wonder difference between `STARTING` and `EXECUTING`.

Anyway we should use a standardized vocabulary throughout API comments and all other documentation. How do we explain the concept of plan start/end and aos/los in the plans API?

Reply:
There is actually a small disconnect when you consider the existing API.

The messages sent on this monitoring stream are from the ground station point of view, whereas they are being received by the satellite operator. The most obvious case is for 'plan start'. For the ground station, plan start is a few minutes before AOS but we haven't communicated that notion of plan start to the satellite operator before.

There is also an existing status enum in the satellite operator 'list plans' API:

    // The plan is reserved for execution in the future at `aos_time`.
    RESERVED = 0;
    // The plan is currently executing and will end in the future at `los_time`.
    EXECUTING = 1;
    // The plan already finished executing successfully.
    SUCCEEDED = 2;
    // The plan failed to execute correctly.
    FAILED = 3;
    // The plan was canceled. This can be explicitly by calling `CancelPlan` or implicit due to
    // cancellation by the ground station in the plan (e.g., due to bad weather).
    CANCELED = 4;
    // The plan completed execution and telemetry data is being processed.
    PROCESSING = 5;
 
Since that enum was written, a few decisions were made:

- SUCCEEDED should become COMPLETED
- PROCESSING no longer needs to be returned

The comments for this existing enum need to be updated to refer to 'start_time' and 'end_time' instead of 'aos_time' and 'los_time'. Let me take care of that in a new PR.

Overall, I think that changing STARTED to PREPARING may be clearer to differentiate the GS meaning of plan start from the one we already use in the satellite operator API.